mix.exs

```diff
@@ -46,4 +46,4 @@ defmodule Nerves.Firmware.HTTP.Mixfile do
         { :exjsx, "~> 3.2.0" },
         { :ex_doc, "~> 0.11", only: :dev} ]
     end
-end
\ No newline at end of file
+end
```

Comment:
Same here. It's probably worth a commit to clean up line endings, but it would be better to have a clean commit here. When you `git rebase -i`, if you amend the commit and remove mix.exs from the changes completely, you'll be rid of the spurious line ending addition. If you want to move the line ending change to a separate commit, that would be fine as well.